inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -1,8 +1,12 @@
 """Protein solvation scenario."""
+
+from absl import logging
+
 from functools import singledispatchmethod
```

Comment:
To follow Google style, abs should come after shutil, since it is not a root package of Python.